configs/common/packages.yaml

```diff
@@ -58,12 +58,12 @@
         - any_of: ['fflags="-fp-model precise" cxxflags="-fp-model precise"']
           when: "%intel"
           message: "Extra ESMF compile options for Intel"
-        - any_of: ['']
-          when: "%gcc"
-          message: "Extra ESMF compile options for GCC"
-        - any_of: ['']
-          when: "%apple-clang"
-          message: "Extra ESMF compile options for GCC"
+        #- any_of: ['']
+        #  when: "%gcc"
+        #  message: "Extra ESMF compile options for GCC"
+        #- any_of: ['']
+        #  when: "%apple-clang"
+        #  message: "Extra ESMF compile options for GCC"
```

Comment:
What's the story here? Are these lines just unnecessary?

Reply:
Yes, they are unnecessary.

Reply:
The only compiler that needs extra flags is intel, and that item only gets added "when: "%intel" (should also have oneapi in the future)